jsonschema-hyperschema.xml

```diff
@@ -1137,6 +1137,209 @@ for varname in templateData:
+                    for fragments, the context pointer is necessary to fully describe the
+                    context.  Its default behavior is to be the same as the attachement pointer.
+                </t>
+            </section>
+            <section title="Individually identified resources">
+                <t>
+                    Let's add "things" to our system, starting with an individual thing:
+                </t>
+                <figure>
+                    <artwork>
+<![CDATA[{
+    "$id": "https://schema.example.com/thing",
+    "$schema": "http://json-schema.org/draft-07-wip/hyper-schema#",
+    "base": "http://api.example.com",
+    "type": "object",
+    "required": "data",
+    "properties": {
+        "id": {"$ref": "#/definitions/id"},
+        "data": true
+    },
+    "links": [
+        {
+            "rel": "self",
+            "href": "things/{id}",
+            "hrefRequired": "id",
+            "targetSchema": {"$ref": "#"}
+        }
+    ],
+    "definitions": {
+            "type": "integer",
+            "minimum": 1,
+            "readOnly": true
+        }
+    }
+}]]>
+                    </artwork>
+                </figure>
+                <t>
+                    Our "thing" has a server-assigned id, which is required in order to
+                    construct the "self" link.  It also has a "data" field which can be
+                    of any type.  The reason for the "definitions" section will be clear
+                    in the next example.
+                </t>
+                <t>
+                    Note that "id" is not required by the validation schema, but is required
+                    by the self link.  This makes sense: a "thing" only has a URI if it has
+                    been created, and the server has assigned an id.  However, you can use
+                    this schema with an instance containing only the data field, which allows
+                    you to validate "thing" instances that you are about to create.
+                </t>
+                <t>
+                    Let's add a link to our entry point schema that lets you jump directly
+                    to a particular thing if you can supply it's id as input.  To save space,
+                    only the new LDO is shown.  Unlike "self" and "about", there is no
+                    IANA-registered relationship about hypothetical things, so an extension
+                    relationship is defined using the
+                    <xref target="RFC4151">"tag:" URI scheme</xref>:
+                </t>
+                <figure>
+                    <artwork>
+<![CDATA[{
+    "rel": "tag:rel.example.com,2017:thing",
+    "href": "things/{id}",
+    "hrefSchema": {
+        "required": ["id"],
+        "properties": {
+            "id": {"$ref": "thing#/definitions/id"}
+                "type": "integer",
+                "minimum": 1
```

Comment:
`"type"` and `"minimum"` should not be there I guess?

Reply:
This is what I get for deciding, fairly late at night, "meh, this is far enough along for feedback" and just pushing it with only a cursory proofread.  Sorry about that :-/